### ISSUE TITLE
Unable to unlearn transmutation mastery

### ISSUE THREAD

**Issue by sanvenganza2** [dalaranwow/dalaran-wow#5701]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

Character name (me) : Alredydead

When you go to an NPC called Zarevhi after learning transmutation mastery by doing the quest. You must be able to unlearn it from him by paying 150 gold. I am unable to unlearn it. There is no option there. The NPC appears blank.

**Expected behaviour**: Tell us what should happen instead

https://www.wowhead.com/quest=10907/master-of-transmutation
After doing this quest, the player should be able to pay 150 gold to unlearn it.

https://wotlk-twinhead.twinstar.cz/?spell=41565

**Steps to reproduce the problem**:

1. Do que transmutation mastery quest
2. After doing it, try unlearning it on the same NPC
3. 

**Include proofs for this behaviour**

https://wotlk-twinhead.twinstar.cz/?spell=41565
![image](https://user-images.githubusercontent.com/48453990/54151870-20c4b680-443c-11e9-95b1-a184a90c29f3.png)
![image](https://user-images.githubusercontent.com/48453990/54151901-3508b380-443c-11e9-8833-2172b587a746.png)

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by mercred** [comment]
Was able to unlearn it on my alch successfully multiple times.
